test/integration/config-devtool-dev/test/index.test.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-env jest */
+
+import { join } from 'path'
+import { launchApp, findPort, killApp, waitFor } from 'next-test-utils'
+
+jest.setTimeout(1000 * 30)
+
+describe('devtool set in developmemt mode in next config', () => {
+  it('should warn & revert when a devtool is set in development mode', async () => {
```

Comment:
We need to verify the source map type was reverted somehow. Maybe we can try to set it to `false` (off) then verify they're present?

Reply:
Thanks for the help! I took a similar approach as this. When running tests the `devtool` is set to `false` by default. So instead I changed it to `source-map` which outputs actual `.map` files and checked to make sure those did not exist in the `.next/static` folder.